packages/workflow/src/internals.ts

```diff
@@ -657,6 +789,30 @@ export class Activator implements ActivationHandler {
     });
   }
 
+  private acceptUpdate(updateId: string): void {
+    const protocolInstanceId = updateId;
+    this.pushCommand({
+      updateResponse: { protocolInstanceId, accepted: this.payloadConverter.toPayload(null) },
```

Comment:
Why do we send back a null payload? Seems like `accepted` is just expecting an empty object; a payload is not the cheapest thing we could write there...

Reply:
Also wondering about that.
I didn't know that accepted had a payload associated with it, you could simply use an empty object if this has any significance:

```suggestion
      updateResponse: { protocolInstanceId, accepted: {} },
```